### ISSUE TITLE
Instances: Settings tab

### ISSUE THREAD

**Issue by benjaminleonard** [oxidecomputer/console#1992]
Presently nothing in here is editable, but might allow for easier discovery of the delete button and in the future to accommodate name changes

<img src="https://uploads.linear.app/b7545512-bb0c-4743-adc4-e5797212a353/2117b91b-da4a-47c0-aa6d-53576645663e/1998123d-c2ca-48a4-aafd-f224d8145e88?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2I3NTQ1NTEyLWJiMGMtNDc0My1hZGM0LWU1Nzk3MjEyYTM1My8yMTE3YjkxYi1kYTRhLTQ3YzAtYWE2ZC01MzU3NjY0NTY2M2UvMTk5ODEyM2QtYzJjYS00OGE0LWFhZmQtZjIyNGQ4MTQ1ZTg4IiwiaWF0IjoxNzA5MDU1MzMwLCJleHAiOjMzMjc5NjE1MzMwfQ.erLtzUG5gz7cV7DpDqfxhadRVr7ppG3aTYg4OmXyHU0 " alt="image.png" width="809" height="542" />

**Comment by charliepark** [comment]
As deleting can only happen if the instance is stopped, we'll probably want a means of doing that on this view? Also Starting, Rebooting?

**Comment by askfongjojo** [comment]
> We have a settings tab, but ID, name, and delete aren't on there. However, now that we have actual settings, maybe that all feels less necessary.

Agreed, more settings (e.g., CPU platform, virtual platform) are coming. Let's mark this close.
